io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,212 @@
+{
+    "app-id": "io.github.rinigus.OSMScoutServer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "osmscout-server",
+    "finish-args": [
+        "--filesystem=host",
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network"
+    ],
+    "rename-icon": "osmscout-server",
+    "cleanup": ["/include", "/lib/mapnik", "/lib/pkgconfig",
+                "/lib/*.a", "/share/doc", "/share/info", "/share/man",
+                "/share/stylesheets",
+                "/bin/cct", "/bin/cs2cs", "/bin/geod", "/bin/gie", "/bin/invgeod", "/bin/invproj",
+                "/bin/kc*", "/bin/libpostal_data", "/bin/lua", "/bin/luac",
+                "/bin/mapnik-config", "/bin/marisa*",
+                "/bin/marisa-predictive-search", "/bin/marisa-reverse-lookup", "/bin/nad2bin", "/bin/proj",
+                "/bin/protoc", "/bin/shapeindex", "/bin/valhalla*"
+               ],
+    "modules": [
+        {
+            "name": "marisa",
+            "subdir": "marisa-trie",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rinigus/pkg-marisa.git",
+                    "commit": "51e65af758de8e00837b6c120c57eb3fa06998e6"
+                }
+            ]
+        },
+        {
+            "name": "libpostal",
+            "subdir": "libpostal",
+            "config-opts": [
+                "--datadir=/app/shared/libpostal/data",
+                "--disable-data-download"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rinigus/pkg-libpostal.git",
```

Comment:
Use upstream sources.

Reply:
Upstream has a bug that has been not fixed due to the absence of maintainer. So, I will use my own fork for libpostal